shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -713,35 +646,30 @@ - (void)setUpChannels {
 
 - (void)maybeSetupPlatformViewChannels {
   if (_shell && self.shell.IsSetup()) {
-    FlutterPlatformPlugin* platformPlugin = _platformPlugin.get();
-    [_platformChannel.get() setMethodCallHandler:^(FlutterMethodCall* call, FlutterResult result) {
-      [platformPlugin handleMethodCall:call result:result];
```

Comment:
Here and a couple more below -- notice that I've moved these to use `weakSelf.platformPlugin`. 

Technically, this does very slightly change the semantics here, but for the better since if `self` were to be cleaned up, these will all devolve into no-ops.